install/ui/src/freeipa/user.js

```diff
@@ -361,6 +361,41 @@ return {
                     fields: [
                         { $type: 'multivalued', name: 'carlicense' }
                     ]
+                },
+                {
+                    name: 'trust attributes',
+                    label: '@i18n:objects.trust_attributes.title',
+                    fields: [{
+                            name: 'ipantlogonscript',
+                            tooltip: {
+                                title: '@i18n:objects.trust_attributes.logonscript_tooltip'
```

Comment:
It is better to have these names exactly same as LDAP attributes, so `objects.trust_attributes.ipantlogonscript_tooltip`, to avoid confusion. Same in other places.